lte/gateway/python/magma/pipelined/policy_converters.py

```diff
@@ -155,6 +154,28 @@ def flip_flow_match(match):
     )
 
 
+def get_flow_ip_dst(match):
+    ip_dst = getattr(match, 'ip_dst', None)
+    if ip_dst is None:
+        return
+    decoded_ip = ip_dst.address.decode('utf-8')
+
+    if ip_dst.version == IPAddress.IPV4:
+        return decoded_ip
+    else:
+        return None
+
+
+def ip_address_to_str(ipaddr: IPAddress):
+
+    decoded_ip = ipaddr.address.decode('utf-8')
+
+    if ipaddr.version == IPAddress.IPV4:
+        return decoded_ip
+    else:
+        return None
```

Comment:
either rename function to ipv4_address_to_str or return decoded_ip on both ip versions?